tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -442,6 +442,105 @@ def test_performance_homepage_query(self):
             assert field_meta["failure_rate()"] == "percentage"
             assert field_meta["failure_count()"] == "integer"
 
+    def test_user_misery_and_team_key_sort(self):
+        self.features["organizations:use-metrics-layer"] = True
```

Comment:
Can you remove this? The `OrganizationEventsMetricsEnhancedPerformanceEndpointTestWithMetricLayer` class already adds this automatically

Reply:
Sure! Sorry for that